src/database.rs

```diff
@@ -50,6 +50,9 @@ pub trait AstarteDatabase {
 
     /// Removes all saved properties from the database
     async fn clear(&self) -> Result<(), AstarteError>;
+
+    /// Retrieves all property values in the database, together with their interface name, path and major version
+    async fn load_all_props(&self) -> Result<Vec<(String, String, Vec<u8>, i32)>, AstarteError>;
```

Comment:
I think it's better to make a struct instead of this giant tuple
```rust
struct DbProp{
        interface: String,
        path: String,
        value: Vec<u8>,
        interface_major: i32,
}
...

async fn load_all_props(&self) -> Result<Vec<DbProp>, AstarteError>;
```

The struct is also useful for querying: 
```rust
sqlx::query_as!(DbProp, "select ...
```